src/rgw/rgw_common.h

```diff
@@ -385,7 +385,7 @@ class RGWEnv;
 class RGWConf {
   friend class RGWEnv;
 protected:
```

Comment:
Drop `protected` as it is not being used and `RGWEnv` is a friend of `RGWConf`.

Reply:
protected has been dropped in patch  #17432

Reply:
@C2python Yes. As your response got delayed, I have submitted that in my patch. I will continue the review of this. Thanks.